packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/search-input/index.tsx

```diff
@@ -2,35 +2,36 @@ import { observer } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { useDBotStore } from 'Stores/useDBotStore';
 import React from 'react';
+import debounce from 'lodash.debounce';
 
+const DEBOUNCE_INTERVAL_TIME = 1000;
```

Comment:
```suggestion
const DEBOUNCE_INTERVAL_TIME = 300;
```
@rupato-deriv lets reduce the debounce interval to 300ms